kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -127,9 +126,8 @@ class TunnelService : VpnService() {
 
                 val route = Cidr(addr, prefix)
                 tunnelRoutes.add(route)
-                val fd = buildVpnService().establish()?.detachFd() ?: -1
-                protect(fd)
```

Comment:
I don't believe this protect was doing anything; the Android docs say this should be used explicitly on a socket: https://developer.android.com/reference/android/net/VpnService#protect(java.net.Socket)